dprint.h

```diff
@@ -340,49 +353,27 @@ static inline char *dp_log_level_str(int log_level)
 		#define LM_GEN2( _facility, _lev, fmt, ...) \
 			do { \
 				if (is_printable(_lev)){ \
+					DISABLE_FORMAT_WARNING; \
 					switch(_lev){ \
 					case L_CRIT: \
-						dprint(_lev, _facility, NULL, NULL, \
-							DP_CRIT_PREFIX fmt, "%s" DP_CRIT_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
-						break; \
 					case L_ALERT: \
-						dprint(_lev, _facility, NULL, NULL, \
-							DP_ALERT_PREFIX fmt, "%s" DP_ALERT_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
-						break; \
 					case L_ERR: \
-						dprint(_lev, _facility, NULL, NULL, \
-							DP_ERR_PREFIX fmt, "%s" DP_ERR_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
-						break; \
 					case L_WARN: \
-						dprint(_lev, _facility, NULL, NULL, \
-							DP_WARN_PREFIX fmt, "%s" DP_WARN_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
-						break; \
 					case L_NOTICE: \
-						dprint(_lev, _facility, NULL, NULL, \
-							DP_NOTICE_PREFIX fmt, "%s" DP_NOTICE_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
-						break; \
 					case L_INFO: \
-						dprint(_lev, _facility, NULL, NULL, \
-							DP_INFO_PREFIX fmt, "%s" DP_INFO_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
-						break; \
 					case L_DBG: \
 						dprint(_lev, _facility, NULL, NULL, \
-							DP_DBG_PREFIX fmt, "%s" DP_DBG_TEXT fmt, fmt, \
-							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
+							"%.0s%.0s%.0s" fmt, "%.0s" fmt, fmt, \
+							dp_time(), dp_my_pid(), log_prefix __VA_ARGS__); \
 						break; \
 					default: \
 						if (_lev > L_DBG) \
 							dprint(_lev, _facility, NULL, NULL, \
-								DP_DBG_PREFIX fmt, "%s" DP_DBG_TEXT fmt, fmt, \
-								dp_time(), dp_my_pid(), log_prefix __VA_ARGS__) \
+								"%.0s%.0s%.0s" fmt, "%.0s" fmt, fmt, \
+								dp_time(), dp_my_pid(), log_prefix __VA_ARGS__); \
```

Comment:
I kept the unused vararg parameters (`dp_time()`, `dp_my_pid()` and `log_prefix`) in `dprint(…)` and skipped logging them with the format specifiers. This helped to keep the code changes at minimum and to avoid modifying code used by other logging functions, reducing the chance of mistakes and future conflicts. This is a temporary disablement in short-term, so for now, I think this is fine.